pypesto/objective/amici/amici.py

```diff
@@ -214,6 +214,9 @@ def __init__(
         # `set_custom_timepoints` method for more information.
         self.custom_timepoints = None
 
+        # Initialize the dictionary for saving of inner parameters.
+        self.inner_parameters = {}
```

Comment:
```suggestion
        self.inner_parameters: Dict[str,float] = dict()
```

to get type annotations right, also not intended to initalize as empty set, right?

Reply:
Yeah typo 😅

Reply:
Oh actually flake8 doesn't like dict()
<img width="823" alt="image" src="https://user-images.githubusercontent.com/85789271/236403947-e35fccfc-6168-45c6-8840-f266b4d73cca.png">